WIP: Implement CRI stats collection (#7796)
cri-o/cri-o: docs/crio.8.md

[--cni-config-dir]=[value]
[--cni-default-network]=[value]
[--cni-plugin-dir]=[value]

> [--collection-period]=[value]

Reviewer: Ditto.

[--hostnetwork-disable-selinux]
[--image-volumes]=[value]
[--imagestore]=[value]

> [--included-pod-metrics]=[value]

Reviewer: Ditto.

**--imagestore**="": Store newly pulled images in the specified path, rather than the path provided by --root.

> **--included-pod-metrics**="": A list of included pod metrics. Specify the names of the metrics to be included in this list.

Reviewer: ```suggestion
**--included-pod-metrics**="": A list of pod metrics names to include.
```

**--signature-policy-dir**="": Path to the root directory for namespaced signature policies. Must be an absolute path. (default: "/etc/crio/policies")

> **--stats-collection-period**="": The number of seconds between collecting pod and container stats. If set to 0, the stats are collected on-demand instead. This option is deprecated and will be removed in the future. (default: 0)

Reviewer: ```suggestion
**--stats-collection-period**="": The number of seconds between collecting pod and container stats. If set to 0, the stats are collected on-demand instead. DEPRECATED: This option will be removed in the future. (default: 0)
```
